CONTRIBUTING.md

```diff
@@ -47,6 +47,15 @@ specialized errors representing the found bugs. The function can also guide the
 towards producing interesting inputs that trigger the bugs of interest. You can have a 
 look at [the existing bug detectors](https://github.com/CodeIntelligenceTesting/gofuzz/tree/main/sanitizers/detectors).
 for more details.
+
+### Dependencies of the `sanitizers` package
+**ALL** transitive dependencies of the `sanitizers` package must be excluded from 
+instrumentation by **gofuzz**. Failing to do so may result in import cycles 
+and cause build failures. This can happen if a dependency `foo` calls a function/method
```

Comment:
... **can** cause build failures. A possible scenario of this would be if a dependency `foo`...

Reply:
Adjusted